src/Plotting.jl

```diff
@@ -903,4 +903,126 @@ function PlotStressTime_0D(
     end
 
     return fig, ax, Tau_II_MPa, t_vec
+
```

Comment:
watch out, compilation is broken because you removed and `end` here to close the function above